app.js

```diff
@@ -45,6 +45,9 @@ app.get('/hangouts/:encounterId', (request, response) => {
 	const key = datastore.key(['Encounter', request.params.encounterId]);
 	datastore.get(key).then(entity => {
 		if (entity) {
+			if(settings.debugLogging) {
```

Comment:
Space before the '(' [here and elsewhere]

Reply:
As an aside, a tiny module that provides 
function debug('foo') that does this check might be nicer than duplicating this check all over

Reply:
Good idea, I will make the change.

Reply:
Added new function debugLog.